addons/web_editor/static/src/components/media_dialog/file_selector.xml

```diff
@@ -57,9 +57,10 @@
             validateUrl="validateUrl"
             multiSelect="props.multiSelect"/>
         <t t-call="{{ constructor.attachmentsListTemplate }}"/>
-        <div name="load_more_attachments" class="mt-4 text-center mx-auto o_we_load_more" t-ref="load-more-button">
-            <button t-if="canLoadMore" class="btn btn-primary o_load_more" type="button" t-on-click="handleLoadMore">Load more...</button>
-            <div t-if="hasContent and !canLoadMore" class="mt-4 o_load_done_msg">
+        <div name="load_more_attachments" class="position-fixed pt-3 text-center mx-auto o_we_load_more" t-ref="load-more-button">
+            <div t-if="canScroll" class="d-flex align-items-center mx-auto btn btn-primary rounded-circle fa fa-chevron-down o_scroll_attachments" t-on-click="handleScrollAttachments"/>
+            <button t-if="canLoadMore and !canScroll" class="btn btn-primary o_load_more" type="button" t-on-click="handleLoadMore">Load more...</button>
```

Comment:
I'd split the lines because there is 2 lines in this PR being over 150 char long
```suggestion
            <button t-if="canLoadMore and !canScroll" class="btn btn-primary o_load_more" type="button" t-on-click="handleLoadMore">
                Load more...
            </button>
```

```suggestion
            <button t-if="canLoadMore and !canScroll" class="btn btn-primary o_load_more" type="button"
                t-on-click="handleLoadMore">
                Load more...
            </button>
```

But I don't think we have good practices / guidelines in Odoo since this is an endless debate between those who want to indent the line return attributes to the same indent as the first attribute from the line above (ver bad imo especially with owl where your component tag could be very long, you would just lose space for no reason) and those who want to just add a simple tab (which then has the same indent as inner element which is not ideal either).

So feel free to ignore this one (just mark it as resolved)

Reply:
I will keep it like this for these reasons:
- they are not the longest lines of the file :stuck_out_tongue: 
- the "attributes" are not really important (classes, onClick, ref) If they were props or data-attributes, I would agree with you to split the lines :slightly_smiling_face:

Reply:
It's not a dictatorship so I ear you :D 
But FWIW:
- Longest or not, it's nowhere near a good argument, it's like "I killed him but I didn't cut his head off, someone did that, so I am not the worst killer", still a killer :D 
- Attributes important or not, it's just about not having a line that will either (depending of your IDE config):
  - Line return (never horizontal scroll)
  - Hide some part of the line off screen (behind a horizontal scroll)

Most people don't care about this one and it's not a really big deal so I am fine with it, even if I am sad about it (kidding.. I guess :D).